backend/data/skills.yaml

```diff
@@ -450,24 +534,30 @@ requirements:
     Logistics Cruisers:
       min: 5
     Sensor Linking:
-      min: 3
-      elite: 4
-    Remote Armor Repair Systems:
-      min: 5
+      min: 4
+      elite: 5
     Magnetometric Sensor Compensation:
       elite: 4
     Gallente Cruiser:
       min: 5
     Medium Drone Operation:
-      min: 3
+      min: 1
       elite: 5
+    CPU Management:
+      min: 4
+    Power Grid Management:
+      min: 5
     Frequency Modulation:
       elite: 4
     Long Distance Jamming:
       elite: 4
+    Drones:
+      min: 5
+    Long Range Targeting:
```

Comment:
Skill can be removed from ship specific as should be an all logi requirement imo.